tests/conftest.py

```diff
@@ -27,7 +27,8 @@ def pytest_addoption(parser):
 @pytest.fixture
 def backend(backend_name, request):
     if request.config.getoption("--gpu-only"):
-        if backend_name not in ("cupy", "cuquantum"):
+        is_gpu = "cupy" in backend_name or "cuquantum" in backend_name
+        if not is_gpu:
```

Comment:
Would you elaborate more this change's explanation?

I see that you are trying to catch more backends in your net, rather than the sole `cupy` and `cuquantum`. But there should be no further GPU backend in Qibojit beyond these two. Especially with those components in the name. Isn't it?

Reply:
Btw, to avoid (very mild) duplication you could still do the same keeping the former list
```suggestion
        if not any(b in backend_name for b in ("cupy", "cuquantum")):
```

Reply:
Hopefully, it's just doing one of the two things. And, most likely, there has been a transition at some point.

If you wish, you could reinstate the former one, just using the complete name, as `"qibojit-cupy"`. At least, if there is this uncertainty, this could be some safe surface where it could float to.

Reply:
btw I just double checked and here in qibojit the ``backend_name`` is just the platform name, whereas in ``qibo`` it appears to be the complete name.

Reply:
I guess it makes sense, since there is no doubt about the package here.

Then, I would just keep the current convention, and check for strict equality.

Reply:
Yep I did it yesterday.